setup/helm/varadhi/values/controller_values.yaml

```diff
@@ -0,0 +1,240 @@
+# Default values for varadhi.
```

Comment:
This is duplicating the entire values files for each component, that shouldn't be needed. What is the proposed hierarchy ?